### PR TITLE
Lunar Festival Fixes

### PR DESCRIPTION
**Pull request by Oroxzy**
- [x] Respawn Elders from sniffs. 
- [x] Add missing Objects (180755, 180756)
- [x] Fix Cluster Launcher rocket positions.
- [x] Add Moonglade Revelers and Objects to China Lunar event.

**Facts:**

- Calculated Rocket positions for launchers:
- https://docs.google.com/spreadsheets/d/1JJ-e3jW_4tIYoFX3_ORbnPS-evJxezaV-1Y8aABnBME/edit?usp=sharing

### CODE REVIEW

File: src/game/Spells/SpellEffects.cpp [hunk 1]
```diff
@@ -5638,13 +5629,10 @@ void Spell::EffectActivateObject(SpellEffectIndex eff_idx)
             break;
         case GameObjectActions::AnimateCustom0:
             gameObjTarget->SendGameObjectCustomAnim(0);
-            break;
```
Comment (ratkosrb) [inline src/game/Spells/SpellEffects.cpp]: wtf are you doing here, now it will fall through

File: src/game/Spells/SpellEffects.cpp [hunk 2]
```diff
@@ -697,15 +697,6 @@ void Spell::EffectDummy(SpellEffectIndex eff_idx)
                     }
                     return;
                 }
-                case 26471: // Lunar Festival Port Error
-                {
-                    if (!m_caster->IsPlayer())
-                        return;
-
-                    m_caster->RemoveSpellCooldown(26373); // Remove cooldown from Lunar Invititation
```
Comment (ratkosrb) [inline src/game/Spells/SpellEffects.cpp:705]: why, this is needed to not set cooldown when u use it in wrong area

Comment (Oroxzy) [reply src/game/Spells/SpellEffects.cpp:705]: its wrong.. no one sniffed it before me, it just teleports you to the moonglade position

Comment (Oroxzy) [reply src/game/Spells/SpellEffects.cpp:705]: it was just a quick improvisation from myself, because nobody could tell me how it really works

File: src/game/Spells/SpellEffects.cpp (hunk 1 above)
Comment (Oroxzy) [reply src/game/Spells/SpellEffects.cpp]: was just a test, i fixed it via spell_effect_mod

File: sql/migrations/20220205202111_world.sql [hunk 3]
```diff
@@ -248,6 +248,43 @@ INSERT INTO `spell_effect_mod` (`Id`, `EffectMiscValue`, `Comment`) VALUES
...
+UPDATE `spell_scripts` SET `comments`='Lunar Teleport Cap: Ironforge - Port Horde Player to Moonglade' WHERE `id`=26408 AND `comments`='Lunar Teleport Cap: Ironforge - Send Error to Horde Player';
+UPDATE `spell_scripts` SET `comments`='Lunar Teleport Cap: Stormwind - Port Horde Player to Moonglade' WHERE `id`=26406 AND `comments`='Lunar Teleport Cap: Stormwind - Send Error to Horde Player';
+
+INSERT INTO `spell_effect_mod` (`Id`, `EffectIndex`, `EffectImplicitTargetA`, `Comment`) VALUES
+    (26406, 0, 25, 'Fix for Holiday - Lunar Teleport Cap: Stormwind'),
+    (26408, 0, 25, 'Fix for Holiday - Lunar Teleport Cap: Ironforge'),
+    (26409, 0, 25, 'Fix for Holiday - Lunar Teleport Cap: Darnassus'),
+    (26410, 0, 25, 'Fix for Holiday - Lunar Teleport Cap: Undercity'),
+    (26412, 0, 25, 'Fix for Holiday - Lunar Teleport Cap: Thunderbluff'),
+    (26414, 0, 25, 'Fix for Holiday - Lunar Teleport Cap: Orgrimmar');
```
Comment (ratkosrb) [inline sql/migrations/20220205202111_world.sql]: Why is this needed? The spells seemed to already work? I remember i tested your last Lunar Festival PR and there were no problems.

Comment (Oroxzy) [reply sql/migrations/20220205202111_world.sql]: it's not needed, i already removed these mods.

File: sql/migrations/20220205202111_world.sql [hunk 4]
```diff
@@ -278,6 +278,218 @@ INSERT INTO `conditions` (`condition_entry`, `type`, `value1`, `value2`, `value3
 INSERT INTO `conditions` (`condition_entry`, `type`, `value1`, `value2`, `value3`, `value4`, `flags`) VALUES (261, 52, 12478, 12496, 12487, 0, 1);
 
 
+-- 1111:  Not (Source's Entry Is 15882)
+INSERT INTO `conditions` (`condition_entry`, `type`, `value1`, `value2`, `value3`, `value4`, `flags`) VALUES (1111, 16, 15882, 0, 0, 0, 1);
+
+INSERT INTO `spell_script_target` (`entry`, `type`, `targetEntry`, `conditionId`, `inverseEffectMask`, `build_min`, `build_max`) VALUES (26347, 0, 180771, 1111, 0, 5086, 5875);
+INSERT INTO `spell_script_target` (`entry`, `type`, `targetEntry`, `conditionId`, `inverseEffectMask`, `build_min`, `build_max`) VALUES (26347, 0, 180868, 1111, 0, 5086, 5875);
+INSERT INTO `spell_script_target` (`entry`, `type`, `targetEntry`, `conditionId`, `inverseEffectMask`, `build_min`, `build_max`) VALUES (26347, 0, 180850, 1111, 0, 5086, 5875);
```
Comment (ratkosrb) [inline sql/migrations/20220205202111_world.sql:283]: Whats the point of the condition here given that the entry is part of this table? 180850 is not equal to 15882, so what does the condition do?

Comment (Oroxzy) [reply sql/migrations/20220205202111_world.sql:283]: Firework_explaination
